pom.xml

```diff
@@ -22,7 +22,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <!-- remember to change the io.jenkins.tools.bom artifact when changing this -->
-    <jenkins.version>2.319.1</jenkins.version>
+    <jenkins.version>2.359</jenkins.version>
```

Comment:
Needs corresponding changes to the `Jenkinsfile` as in https://github.com/jenkinsci/sshd-plugin/blob/2dc2ea416e3300ad210e3943b471204d562e6d18/Jenkinsfile

Reply:
(You would need write access to test that. If you do not have write access, I can refile as a separate testing-only PR. Ideally `github-branch-source` could be instructed by a maintainer somehow to test a given commit.)

Reply:
conveniently I have global admin 😄.

and yes that would be good